utils/README.md

```diff
@@ -11,7 +11,7 @@ suspend.bin is a MacBinary file which should be unpacked and run in M68k Macinto
 
 # Break point commands
 
-```bash
```

Comment:
This should really be a markdown table

```markdown
Command | Effect
--------|:------
`ba [address]` | Add a break point
```

etc

Reply:
OK, I will add a commit.